examples/Dropdown.elm

```diff
@@ -81,21 +81,21 @@ noDate whenText =
 
 
 formatDate : Date -> String
-formatDate date =
-    (toString (Date.day date)) ++ " " ++ (toString (Date.month date)) ++ " " ++ (toString (Date.year date))
+formatDate =
+    Date.format "d MMM y"
 
 
 config : DatePicker.Config
 config =
     let
-        config =
+        config_ =
```

Comment:
https://til.hashrocket.com/posts/vmn6mvmll7-whats-the-trailing-underscore-in-elm-